Dockerfile

```diff
@@ -46,13 +46,13 @@ RUN mkdir -p /prebuilt_docs/teaclave-docs && \
 RUN rm -rf teaclave/build
 
 # Dependencies stage
-FROM node:16.20.2-bullseye AS deps
+FROM node:22-slim AS deps
 WORKDIR /app
 COPY site/package*.json site/
 RUN cd site && npm install
 
 # Final stage
-FROM node:16.20.2-bullseye as teaclave-docs-site
+FROM node:22-slim as teaclave-docs-site
```

Comment:
I am encountering the pip3 install error when `docker build` with the new base-image.
Could you please replace the line with `RUN pip3 install --break-system-packages ghp-import`

```bash
#28 0.697 hint: See PEP 668 for the detailed specification.
------
process "/bin/sh -c pip3 install ghp-import" did not complete successfully: exit code: 1
```